tests/pytests/test_dft_benchmarks.py

```diff
@@ -154,11 +154,12 @@ def name_dft_test(val):
     pytest.param(        'wB97M-V', 0.4544676075, '6-31G'),  # Q-Chem
     pytest.param(          'wB97X', 0.4564711283, '6-31G'),  # Q-Chem
     pytest.param(        'wB97X-D', 0.4575912358, '6-31G', marks=pytest.mark.quick),  # Q-Chem
-    pytest.param(       'wB97X-D3', 0.4570744381, '6-31G', marks=[using("dftd3"), pytest.mark.xfail]),  # Q-Chem  # needs tweaks in LibXC
+    pytest.param(       'wB97X-D3', 0.4570744381, '6-31G', marks=[using("dftd3")]),   # Q-Chem  # needs tweaks in LibXC
```

Comment:
Is "needs tweaks in LibXC" still correct?

Reply:
Nope!